src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -35,6 +35,56 @@
 #include "doctest.h"
 
 namespace triton { namespace perfanalyzer {
+cb::Error
+ReportPrometheusMetrics(const Metrics& metrics)
+{
+  if (metrics.gpu_utilization_per_gpu.size() > 4 ||
+      metrics.gpu_power_usage_per_gpu.size() > 4 ||
+      metrics.gpu_memory_used_bytes_per_gpu.size() > 4 ||
+      metrics.gpu_memory_total_bytes_per_gpu.size() > 4) {
+    std::cout << "Too many GPUs on system to print out individual Prometheus "
```

Comment:
is cout the correct way to print in PA? Or is there a logging macro/class?

Reply:
There is not currently a logging macro/class.